coalib/results/TextPosition.py

```diff
@@ -31,3 +31,27 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    def __le__(self, other):
```

Comment:
no, I wrote `__le__` and `__ge__` to handle
`self.start <= other.end and self.end >= other.start` in TextRange.